legislation_analysis/processing/ner.py

```diff
@@ -48,7 +48,7 @@ def apply_ner(text: str, ner_lst: list) -> list:
 
         Parameters:
             text (str): Chunk of text to apply NER to.
-            ner_dict (dict): Dictionary of Named Entity Recognition (NER) data.
+            ner_lst (list): List of Named Entity Recognition (NER) data.
```

Comment:
NER class changes were made because PyCharm said the wrong types were listed or returned.